tests/pytests/common.py

```diff
@@ -744,15 +743,39 @@ def check_index_info_empty(env, idx, fields, msg="after delete all and gc", dept
     expected_size = getInvertedIndexInitialSize_MB(env, fields, depth=depth+1)
     check_index_info(env, idx, exp_num_records=0, exp_inv_idx_size=expected_size, msg=msg, depth=depth+1)
 
-def downloadFiles(rdbs = None):
+def downloadFiles(env, rdbs=None):
     if rdbs is None:
         return False
 
-    os.makedirs(REDISEARCH_CACHE_DIR, exist_ok=True) # create cache dir if not exists
+    os.makedirs(REDISEARCH_CACHE_DIR, exist_ok=True)  # create cache dir if not exists
     for f in rdbs:
         path = os.path.join(REDISEARCH_CACHE_DIR, f)
         if not os.path.exists(path):
-            subprocess.run(["wget", "--no-check-certificate", BASE_RDBS_URL + f, "-O", path, "-q"])
+            try:
+                subprocess.run(
+                    [
+                        "wget",
+                        "--no-check-certificate",
+                        BASE_RDBS_URL + f,
+                        "-O",
+                        path,
+                        "-q",
+                    ],
+                    check=True,
+                )
+            except subprocess.CalledProcessError as e:
```

Comment:
Exception is thrown only when passing `check=True`? Also what about other exception types, is that possible?